src/pools/abstract-pool.ts

```diff
@@ -222,11 +223,33 @@ export abstract class AbstractPool<
     )
   }
 
+  /** @inheritdoc */
+  public abstract get busy (): boolean
+
+  protected internalGetBusyStatus (): boolean {
+    return (
+      this.promiseMap.size >= this.numberOfWorkers &&
```

Comment:
>== would be better ?

Reply:
> ==

Reply:
The second condition needs to be triggered only if the number of tasks is superior or equal to the number of workers.